tensornetwork/__init__.py

```diff
@@ -6,6 +6,7 @@
 from tensornetwork.visualization.graphviz import to_graphviz
 from tensornetwork import contractors
 from tensornetwork import config
+from mps import mps
```

Comment:
Move the code to `tensornetwork/` and change this to `from tensornetwork.mps import mps`

Reply:
This seems to create a cyclic import (linter is complaining about this)